interface/modules/custom_modules/oe-module-comlink-telehealth/moduleConfig.php

```diff
@@ -5,13 +5,102 @@
  *  package   OpenEMR
  *  link      http://www.open-emr.org
  *  author    Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  author    Jerry Padgett <sjpadgett@gmail.com>
  *  copyright Copyright (c )2021. Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  copyright Copyright (c) 2023 Jerry Padgett <sjpadgett@gmail.com>
  *  license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  *
  */
 
 require_once dirname(__FILE__, 4) . '/globals.php';
 
+use OpenEMR\Common\Csrf\CsrfUtils;
+
 $module_config = 1;
 
-require_once('welcome.php');
+if (!empty($_GET['setup']) ?? null) {
+    /**
+     * Example expected data structure for received data.
+     * $args = ['ctsiVBData' => [
+     * 'registrationUri' => 'https://sginsts.comlinktelehealth.io:28749/CTSIVB/',
+     * 'videoApiUri' => 'https://sginsts.comlinktelehealth.io:22528',
+     * 'ctsiOrgUid' => 'OPEN5C0D4F',
+     * 'ctsiOrgPwd' => 'DD71C92B-834D-4C',
+     * 'ctsiOrgId' => 'OPENEM280D0D']];
+     *
+     * */
+    if (!CsrfUtils::verifyCsrfToken($_GET["csrf_token_form"])) {
+        CsrfUtils::csrfNotVerified();
+    }
+
+    $content = trim(file_get_contents("php://input"));
+    $credentials = json_decode($content, true);
+
+    $items['comlink_telehealth_registration_uri'] = $credentials['registrationUri'];
+    $items['comlink_telehealth_video_uri'] = $credentials['videoApiUri'];
+    $items['comlink_telehealth_user_id'] = $credentials['ctsiOrgUid'];
+    $items['comlink_telehealth_user_password'] = $credentials['ctsiOrgPwd'];
+    $items['comlink_telehealth_cms_id'] = $credentials['ctsiOrgId'];
```

Comment:
do any of these get saved in encrypted format (pretty sure the password was on the prior code reviews of this module, but don't recall)?

Reply:
yep you're right. thanks for catching.